pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```diff
@@ -109,20 +101,12 @@ in {
 
   # the dontHaddock is due to a GHC panic. might be this bug, not sure.
   # https://gitlab.haskell.org/ghc/ghc/-/issues/21619
-  #
-  # We need >= 1.1.2 for ghc-9.4 support, but we don't have 1.1.x in
-  # hackage-packages.nix
-  hedgehog = doDistribute (dontHaddock super.hedgehog_1_2);
-  # tasty-hedgehog > 1.3 necessary to work with hedgehog 1.2:
-  # https://github.com/qfpl/tasty-hedgehog/pull/63
-  tasty-hedgehog = self.tasty-hedgehog_1_4_0_1;
+  hedgehog = dontHaddock super.hedgehog;
 
   # https://github.com/dreixel/syb/issues/38
   syb = dontCheck super.syb;
 
   splitmix = doJailbreak super.splitmix;
```

Comment:
`stackage.yaml` has v0.1.0.4 which even allows GHC 9.6. So this shouldn't be needed for GHC 9.4.